package.json

```diff
@@ -4,15 +4,10 @@
   "description": "Prototyping environment for the Amsterdam Design System",
   "name": "design-system-prototypes",
   "private": true,
-  "engines": {
-    "node": "^20",
-    "npm": "^8",
```

Comment:
I wouldn't remove these 2.

Different node versions create different lockfiles, which can get very annoying.
Specifying an npm version also makes sense I think, because different npm versions support different features. The npm features we use are probably not supported by all versions.

Reply:
Yes, I expected them to be useful, but I got some errors I didn’t understand. Must have something else, so I’ve restored all three now.